content-full-no.php

```diff
@@ -11,16 +11,16 @@
 
 		<div class="content-page">
 		
-			<?php if (has_post_thumbnail()): ?>
+			<?php if ( has_post_thumbnail() ) : ?>
 			<div class="featuredImage">
-				<?php echo the_post_thumbnail(700, 300); ?>
+				<?php echo the_post_thumbnail( 700, 300 ); ?>
 			
 			</div>	
 			<?php endif; ?>
 			
 			
 			<div class="entry-content">
-				<?php if (!$isRoot): ?>
+				<?php if ( ! $isRoot ) : ?>
```

Comment:
The original flags for line 23 of content-full-no.php were:
```
 23 | ERROR | [x] No space after opening parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis)
 23 | ERROR | [x] Expected 1 space before "!"; 0 found
    |       |     (WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore)
 23 | ERROR | [x] Expected 1 space after "!"; 0 found
    |       |     (WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter)
 23 | ERROR | [ ] Variable "isRoot" is not in valid snake_case format
    |       |     (WordPress.NamingConventions.ValidVariableName.NotSnakeCase)
 23 | ERROR | [x] No space before closing parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis)
 23 | ERROR | [x] Expected 1 space after closing parenthesis; found 0
    |       |     (Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis)
 23 | ERROR | [x] Space between opening control structure and T_COLON is
    |       |     required
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBetweenStructureColon)
```

The third of these argues that there should be a space between operator and term. This is also laid out in the PHP coding standards on Wordpress.org: 

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage